lighthouse-core/gather/gatherers/dobetterweb/js-vulnerable-libraries.js

```diff
@@ -0,0 +1,96 @@
n:"vue",url:"http://vuejs.org/",test:function(win){if(win.Vue&&win.Vue.nextTick){return{version:win.Vue.version||UNKNOWN_VERSION}}return false}},Two:{icon:"two",url:"https://jonobr1.github.io/two.js",test:function(win){if(win.Two&&win.Two.Utils){return{version:win.Two.Version||UNKNOWN_VERSION}}return false}},Brewser:{icon:"brewser",url:"http://handcraftedldn.github.io/brewser/",test:function(win){if(win.BREWSER&&win.BREWSER.ua){return{version:BREWSER.VERSION||UNKNOWN_VERSION}}return false}},"Material Design Lite":{icon:"mdl",url:"http://www.getmdl.io/",test:function(win){if(win.componentHandler&&win.componentHandler.upgradeElement){return{version:UNKNOWN_VERSION}}return false}},"Kendo UI":{icon:"kendoui",url:"https://github.com/telerik/kendo-ui-core",test:function(win){if(win.kendo&&win.kendo.View&&win.kendo.View.extend){return{version:win.kendo.version||UNKNOWN_VERSION}}return false}},"Matter.js":{icon:"matter-js",url:"http://brm.io/matter-js/",test:function(win){if(win.Matter&&win.Matter.Engine){return{version:UNKNOWN_VERSION}}return false}},Riot:{icon:"riot",url:"http://riotjs.com/",test:function(win){if(win.riot&&win.riot.mixin){return{version:win.riot.version||UNKNOWN_VERSION}}return false}},"Sea.js":{icon:"icon_48",url:"http://seajs.org/",test:function(win){if(win.seajs&&win.seajs.use){return{version:win.seajs.version||UNKNOWN_VERSION}}return false}},"Moment.js":{icon:"momentjs",url:"http://momentjs.com/",test:function(win){if(win.moment&&(win.moment.isMoment||win.moment.lang)){return{version:win.moment.version||UNKNOWN_VERSION}}return false}},"Moment Timezone":{icon:"momentjs",url:"http://momentjs.com/timezone/",test:function(win){if(win.moment&&win.moment.tz){return{version:win.moment.tz.version||UNKNOWN_VERSION}}return false}},ScrollMagic:{icon:"scrollmagic",url:"http://scrollmagic.io/",test:function(win){if(win.ScrollMagic&&win.ScrollMagic.Controller){return{version:ScrollMagic.version||UNKNOWN_VERSION}}return false}},SWFObject:{icon:"icon_48",url:"https://github.com/swfobject/swfobject",test:function(win){if(win.swfobject&&win.swfobject.embedSWF){return{version:win.swfobject.version||UNKNOWN_VERSION}}else if(win.deconcept&&win.deconcept.SWFObject){return{version:UNKNOWN_VERSION}}return false}},FlexSlider:{icon:"icon_48",url:"https://woocommerce.com/flexslider/",test:function(win){var jq=win.jQuery||win.$||win.$jq||win.$j;if(jq&&jq.fn&&jq.fn.jquery&&jq.flexslider){return{version:UNKNOWN_VERSION}}return false}},SPF:{icon:"icon_48",url:"https://youtube.github.io/spfjs/",test:function(win){if(win.spf&&win.spf.init){return{version:UNKNOWN_VERSION}}return false}},"Numeral.js":{icon:"icon_48",url:"http://numeraljs.com/",test:function(win){if(win.numeral&&win.isNumeral){return{version:win.numeral.version||UNKNOWN_VERSION}}return false}},"boomerang.js":{icon:"icon_48",url:"https://soasta.github.io/boomerang/doc/",test:function(win){if(win.BOOMR&&win.BOOMR.utils&&win.BOOMR.init){return{version:win.BOOMR.version||UNKNOWN_VERSION}}return false}}};
```

Comment:
take a look at how we do the accessibility gatherer. we `readFileSync` the axe.min.js from the axe-core node module.

this way, versioning is pretty straightfoward as its just a node module dep bump.